lib/evaluate/variable.cc

```diff
@@ -362,6 +348,12 @@ int Component::Rank() const {
   return base().Rank();
 }
 
+int NamedEntity::Rank() const {
+  return std::visit(common::visitors{[](const Symbol *s) { return s->Rank(); },
+                        [](const Component &c) { return c.Rank(); }},
```

Comment:
If you add a comma after the second lambda, clang-format won't make such a mess of this.